scripts/ci/collect-logs.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+
+user_help () {
+    echo "Collects all logs from the given namespace"
+    echo "options:"
+    echo "-n,   --namespace          The namespace the logs should be collected from."
+    echo "-h,   --help               To show this help text"
+    echo ""
+    exit 0
+}
+
+read_arguments() {
+    if [[ $# -lt 2 ]]
+    then
+        user_help
+    fi
+
+    while test $# -gt 0; do
+           case "$1" in
+                -h|--help)
+                    user_help
+                    ;;
+                -n|--namespace)
+                    shift
+                    NAMESPACE=$1
+                    shift
+                    ;;
+                *)
+                   echo "$1 is not a recognized flag!" >> /dev/stderr
+                   user_help
+                   exit -1
+                   ;;
+          esac
+    done
+}
+
+
+start_collecting_logs() {
+    if [[ -n ${ARTIFACT_DIR} ]]; then
```

Comment:
```suggestion
    if [[ -d ${ARTIFACT_DIR} ]]; then
```

Reply:
my intention was to check only if the `ARTIFACT_DIR` env var is set or not. It doesn't care about the existence - if it doesn't exist, then it would fail in the following code anyway - which means that it would stop this background process (which will also print the message).

Reply:
is this better? a9c18b8

Reply:
Yeah, that makes sense 👍